### DISCUSSION TITLE
Misspelled GitHub Command perma-broke Storage Permissions

### DISCUSSION THREAD

**Discussion by jerrecode** [termux/termux-app#3588] [Q&A] [answered]
Hey do any of you know a fix to this problem:
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Storage permission worked for device storage and termux normaly

could also switch between termux home and 
`//storage/emulated/0/`
normally and was able to use nano, git and github and everything else in a subpath 
`//storage/emulated/0/Code/SomeCodeProject/`

-------------------------------------

but than i made a misstake while pushing to a repo a repo wich was already connected

what happend was i tried to move a few commands back by pressing Arrow Up to save some work and i accidentally used the command to connwct a github repo again and accidantly changed the word origin in the command by misspressing some buttons.

After that i wanted to use git to push my repo as i wanted to do before but it wrote this message:

```
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
```

Then Termux shutdown itsself.

Since then all my storage permissions are gone and i can grant them and it looks like it should work but it doesnt.

looked into my offline code repo with acode but everything is okay there.

**the wrong command kinda looked like that:
basically some misspelled argument**
```
git remote add origingvt git@github.com:username/new_repo
```

**its possible that i misspelled this command instead**
```
git push -u origin mastzf6er
```

since termux shut down my command history is missing a big sectiom.

**Comment by jerrecode** [comment]
**Ended up reinstalling termux.**

_Will escalate that to bugreport even tho I am not shure you can call it a bug because it wasn't caused by normally working commands._

But if anyone finds a proper (no reinstall) sollution you are welcomed to write an answere!

Thank you :D

**Comment by sylirre** [comment, marked as answer]
I would suggest to avoid working with Git repositories at `/storage/emulated/0`. The file system here emulates FAT32/exFAT and is limited in features. People sometimes report weird behavior of `git` here, but really this is not a bug and can't be fixed on Termux side.

Just clone your repos in Termux home. This directory is accessible by file managers and editor applications supporting storage volumes (built-in Android directory viewer and file picker, QuickEdit, FX File Explorer, etc).

**Reply by jerrecode** [reply]
Thanks for the explanation!

Will keep that in mind in the future.
